resources/LayerSelector.css

```diff
@@ -48,7 +48,6 @@
   -webkit-box-shadow: 0 1px 5px rgba(0,0,0,0.4);
   box-shadow: 0 1px 5px rgba(0,0,0,0.4);
   background: #fff;
-  -webkit-border-radius: 5px;
```

Comment:
I didn't actually touch this file by hand. I just assumed that when I ran the grunt stylus plugin it didn't add that prefix. I don't believe that prefix is used anymore.